docs/source/gallery/inference_diagnostics/plot_convergence_dist.py

```diff
@@ -0,0 +1,25 @@
+"""
+# Convergence diagnostics distribution plot
+
+Plot the distribution of ESS and R-hat.
+
+---
+
+:::{seealso}
+API Documentation: {func}`~arviz_plots.plot_ess`
+:::
+"""
+
+from arviz_base import load_arviz_data
+
+import arviz_plots as azp
+
+azp.style.use("arviz-clean")
+
+data = load_arviz_data("radon")
+pc = azp.plot_ess(
```

Comment:
Shouldn't this be `azp.plot_convergence_dist`?

Reply:
Good catch!